playground/AzureStorageEndToEnd/AzureStorageEndToEnd.AppHost/Program.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+var storage = builder.AddAzureStorage("storage").UseEmulator(configureContainer: builder =>
+{
+    // Temporary, we will have an extension method that allows setting persistence and specifying a local
+    // directory (for example).
+    builder.WithVolumeMount(Path.Combine(Environment.CurrentDirectory, "data"), "/data", VolumeMountType.Bind, false);
```

Comment:
We should make volumes relative to the apphost directory.

Reply:
Done.